samples/msgext-search-sso-config/nodejs/bots/teamsMessagingExtensionsSearchAuthConfigBot.js

```diff
@@ -250,7 +250,7 @@ class TeamsMessagingExtensionsSearchAuthConfigBot extends TeamsActivityHandler {
 
                 return {
                     composeExtension: {
-                        type: 'silentAuth',
+                        type: 'auth',
```

Comment:
once there turn `silentAuth` to `auth`, i think there should no return 412 in `onInvokeActivity` line 353

Reply:
Fixed